front/src/components/Map/Map.js

```diff
@@ -87,9 +88,16 @@ export default {
       },
 
       //Markerがクリックされた時に起動する関数
-      markerClickEvent(event){
-        alert(event.target.spot_type);
-        //スポット詳細のダイアログを出す
+      markerClickEvent: async function(event){
+        //For Debudg
```

Comment:
Is this section just for debug/testing? If it is then it should be removed

Reply:
This is for debug, but this is also function to check the review when click the spot.
we cannot test without this function